blueman/main/NetConf.py

```diff
@@ -294,16 +294,14 @@ def __init__(self):
         self.dhcp_handler = None
         self.ipt_rules: List[Tuple[str, str, str]] = []
 
-        self.ip4_address = None
-        self.ip4_mask = None
```

Comment:
Same as before, Pycharm is not happy but flake8 is fine with it. Ok I guess.. :weary: